### ISSUE TITLE
pip install lxml fails, causing build to fail

### ISSUE THREAD

**Issue by wltjr** [Shippable/support#3843]
### Description of your issue:
pip install lxml is failing for some reason. I have not changed anything with regard to my CI configuration since [April](https://github.com/Obsidian-StudiosInc/os-xtoo/commits/master/.travis.yml). Everything is fine on Travis. Everything was fine last Sunday. Since then my [build fails](
https://app.shippable.com/github/Obsidian-StudiosInc/os-xtoo/dashboard) but not at my project. It fails on installing to prepare the environment for my project.

I am not sure if upgrading pip matters or not. Looks like something change in the environment and its causing pip to fail to build dependencies or something. I do not believe it is anything I can address. Please advise, thank you!

**Comment by a-murphy** [comment]
There was a new version of [lxml](https://pypi.python.org/pypi/lxml) released on Sunday.  I found that `lxml` version 4.0.0 will install with a newer version of `pip` and fails with the version of `pip` in the build image you are currently using.

Since you're using a Travis YML file, I'm assuming you don't want to change it.   So I would suggest selecting a newer machine image for your subscription on Shippable, which will cause your builds to use a more recent default build image.  Directions to change the machine image are available [here](http://docs.shippable.com/platform/tutorial/runtime/ami-overview/).  The v5.7.3 and v5.8.2 images have a newer version of `pip` that was able to install `lxml` when I tried it.
